libparsec/src/lib.rs

```diff
@@ -31,3 +31,14 @@ pub use testbed::*;
 pub async fn list_available_devices(config_dir: &std::path::Path) -> Vec<AvailableDevice> {
     libparsec_platform_device_loader::list_available_devices(config_dir).await
 }
+
+pub async fn build_backend_addr(hostname: String, port: Option<u32>, use_ssl: bool) -> BackendAddr {
```

Comment:
```suggestion
pub fn build_backend_addr(hostname: String, port: Option<u32>, use_ssl: bool) -> BackendAddr {
```

Don't need to be async